### ISSUE TITLE
Installing games with options in the installation process will hang the installation

### ISSUE THREAD

**Issue by joalbankluane** [flavioislima/HeroicGamesLauncher#171]
**Describe the bug**
Installing a game for which `legendary` asks for options (such as `Fortnite`) will hang the installation process. The questions asked by `legendary` are stored in the `GamesConfig/<game>.log`. The installation is stuck at 0%.

I solved the issue by copy-pasting the command line shown in `ps waux` in a terminal window and answering the questions to complete the installation normally.

**To Reproduce**
1. Add fortnite (free to play) to your games list.
2. In Heroic, try to install Fortnite.
3. See error

**Expected behavior**
The user should be presented with the questions from legendary, and be allowed to customise the installation accordingly.
As a minimum while the expected behaviour is being implemented, heroic should validate the default choices and complete the installation.

**Desktop (please complete the following information):**
 - OS: Ubuntu Mate
 - Version 20.04
-  Game: Fortnite
-  Wine/Proton version: default (not relevant here)

**Additional context**
Command line used to complete the installation:
`/tmp/.mount_heroicMfbzAg/resources/app.asar.unpacked/build/bin/legendary install Fortnite --base-path <basepath> --max-workers 2 -y` and answering the questions.

**Comment by flavioislima** [comment]
Games with selective downloads, right now are fortnite and cyberpunk, are not supported by heroic right now.

I'll add a note on readme. 

Also, I don't think fortnite works under Linux. Does it?

**Comment by joalbankluane** [comment]
If the list of games using selective download is so small, maybe it would be possible to have some kind of warning beside the installation button, and maybe pointing to the description on the way to go around the problem in the Readme.
Once legendary has completed the installation, I could launch it from heroic. I managed to get to the lobby, but didn't try then to launch a game.
I have now tried. First, the game attempted to update, and the update has been properly completed by Heroic.
Then, indeed, I could get to the lobby and perform a few actions, but trying to start a game crashes it before I could get to play.
However, it might still be worth helping with the initial installation using Heroic, since once that is done, the next step is on wine to support the game (which it might in certain configuration, as it seems now easy cheat is not the problem any more).
I would understand that this gets lower priority to be fixed within Heroic though.

**Comment by flavioislima** [comment]
@joalbankluane I'll try to blacklist those games to show as disabled or with a warning like you said, but I have no idea when I will implement the selective download since, for linux, only Cyberpunk will work. And, for one game, I dont know if it worth the effort. 
Because I'll need to implement a dialog based on legendary output, and then cancel the command and send the command again with the right asset list. At least this was like I imagine to do, so it wont be easy and I think is best to use legendary since is just one game.

**Comment by joalbankluane** [comment]
@flavioislima if it's not too big a problem, maybe you can give the user the command line to apply in a terminal (just print the line instead of executing it), it might be difficult to find out otherwise.
I understand that for only one working game, it's maybe not so much worth it, although I expect that Cyberpunk might be quite popular.
Anyway, thank you for looking into it. It's also OK if you want to close the issue for now. I appreciate very much the work you put into Heroic.
